docs/dev/api_usage_examples/common.py

```diff
@@ -0,0 +1,54 @@
+'''
+common.py
+'''
+
+from enum import Enum
+from time import time
+from datetime import datetime
+
+ROOT = '127.0.0.1'
+API_VERSION = 1
+TOKEN = '93fb3a42-9177-48a0-b07a-e8a7019218f6' # Replace with your dev token.
+USERNAME = 'naiveaiguy' # Replace with the username from which you got the token above.
+AUTH_HEADER = {
+    "Authorization": f"Token {TOKEN}"
+}
+
+class Track(dict):
```

Comment:
Because these are simple API usage examples, I'd prefer to keep boilerplate code to a minimum. Just using a dictionary instead of creating new classes is something we prefer in LB (and other MeB Python projects), so I'd suggest removing this class and just creating a dict listen like

```
EXAMPLE_PAYLOAD = {
    "listen_type": "single",
    "payload": [
        {
            "listened_at": 1486449409,
            "track_metadata": {
                "artist_name": "Kanye West",
                "track_name": "Fade",
                "release_name": "The Life of Pablo"
            }
        }
    ]
}
```

Reply:
My reasoning for this was that it was more semantically understandable in examples. I'm not sure about just using a dictionary.

Reply:
Now that I think about it, both `Track` and `simple_repr` are only used in one file each. I'll simplify those places.

Reply:
Done.